libbindgen/tests/headers/inline_namespace.hpp

```diff
@@ -0,0 +1,11 @@
+// bindgen-flags: --enable-cxx-namespaces -- -std=c++11
+
+namespace foo {
+  inline namespace bar {
+    using Ty = int;
+  };
```

Comment:
Also, what about multiple nested inline namespaces? Seems like another good thing to test.